packages/dashboard/src/components/sidePanel/index.css

```diff
@@ -2,4 +2,5 @@
   height: 100%;
   max-height: 100vh;
   overflow-y: scroll;
+  min-width: 400px;
```

Comment:
Does this cause any issues with the resizable panes? What happens if you resize the side panel down to <400px?

Reply:
It doesn't cause any issue with resizable panes. When it <400px, setting panel will change to vertically scrollable. This was mainly for limiting the setting panel becoming too narrow to pushing contents around.